src/Dotnet.Script/DebugScriptRunner.cs

```diff
@@ -34,9 +33,11 @@ public override Task<TReturn> Execute<TReturn>(ScriptContext context)
 
             foreach (var syntaxTree in compilationContext.Compilation.SyntaxTrees)
             {
+                // https://github.com/dotnet/roslyn/blob/version-2.0.0-beta4/src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.ParsedSyntaxTree.cs#L19
```

Comment:
The problem with all these URLs is that they are not absolute. I.e. what is in line 19 of that file will change over time. Perhaps the file itself will move.

You should link to the specific version at the time of writing, e.g. https://github.com/dotnet/roslyn/blob/eb964d42c8f8e763e4379f1e33c3cf5547692a3f/src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.ParsedSyntaxTree.cs#L19